components/inlinealert/package.json

```diff
@@ -17,13 +17,13 @@
   },
   "peerDependencies": {
     "@spectrum-css/icon": "^3.0.21",
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/tokens": "^1.0.7"
   },
   "devDependencies": {
     "@spectrum-css/button": "^6.0.13",
     "@spectrum-css/component-builder": "^3.2.0",
```

Comment:
Can you remove this dependency, and instead use: `"@spectrum-css/component-builder-simple": "^1.0.0"`?